irohad/simulator/impl/simulator.cpp

```diff
@@ -28,17 +28,28 @@ namespace iroha {
         std::shared_ptr<ametsuchi::TemporaryFactory> factory,
         std::shared_ptr<ametsuchi::BlockQuery> blockQuery,
         std::shared_ptr<model::ModelCryptoProvider> crypto_provider)
-        : validator_(std::move(statefulValidator)),
+        : proposal_subscription_(rxcpp::composite_subscription()),
```

Comment:
No need to call empty constructor, it will be called before constructor body. Same for `verified_proposal_subscription_`